ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/ArrayFieldTemlate.js

```diff
@@ -173,14 +173,15 @@ const DefaultNormalArrayFieldTemplate = (props) => {
 
             {
               (props.uiSchema["ui:description"] || props.schema.description) &&
-              <EnlargedTextTooltip title={props.uiSchema["ui:description"] || props.schema.description}>
+              <CustomTextTooltip backgroundColor="#3C494F" title={props.uiSchema["ui:description"] || props.schema.description}>
                 <IconButton  disableTouchRipple="true" disableRipple="true">
                   <HelpOutlineIcon width="14px" height="14px" fill="black" style={{ marginLeft : '4px' }} />
                 </IconButton>
-              </EnlargedTextTooltip>
+              </CustomTextTooltip>
             }
             {props.rawErrors?.length > 0 && (
-              <EnlargedTextErrorTooltip
+              <CustomTextTooltip
+                backgroundColor="#B32700"
```

Comment:
I swear that we have this color defined in a theme somewhere. Where might that be @Harshit-Dandriyal?

Reply:
@harkiratsm @Harshit-Dandriyal please update